packages/terraform-modules-public/aws/testnet/main.tf

```diff
@@ -0,0 +1,149 @@
+provider "aws" {
+    region = var.region
+}
+
+module "celo_vpc" {
+    source = "./modules/vpc"
+
+    name                = "celo-vpc"
+    cidr_blocks         = var.cidr_blocks
+}
+
+module "celo_bastion_az1" {
+    source            = "./modules/bastion"
+
+    subnet_id         = module.celo_vpc.subnet_ids.az1.public
+    key_pair_name     = var.key_pair_name
+    name              = "celo-bastion-az1"
+    instance_type     = var.instance_types.bastion
+}
+
+module "celo_bastion_az2" {
+    source            = "./modules/bastion"
+
+    subnet_id         = module.celo_vpc.subnet_ids.az2.public
+    key_pair_name     = var.key_pair_name
+    name              = "celo-bastion-az2"
+    instance_type     = var.instance_types.bastion
+}
```

Comment:
Is ssh access to the private instances the goal for the bastion instances? If that is the case, we could also consider aws session manager (https://docs.aws.amazon.com/systems-manager/latest/userguide/session-manager.html). It has some usability issues (like slow pasting to the session), but from a security point of view I find it very nice.

Reply:
Oo, nice. Yes, this is a superior solution security-wise IMO.

I think this is something to consider for a future enhancement. You ok with that?

Reply:
100% agree.